webpack/webpack.config.dev-server.js

```diff
@@ -1,38 +1,43 @@
 var path = require('path');
 var webpack = require('webpack');
-var commonLoaders = require('./common.config').commonLoaders;
-var assetsPath = require('./common.config').assetsPath;
-var publicPath = require('./common.config').publicPath;
+
+var commonConfig = require('./common.config');
+
+var commonLoaders = commonConfig.commonLoaders;
+var publicPath = commonConfig.output.publicPath;
+var externals = commonConfig.externals;
 
 module.exports = {
```

Comment:
Strange, I can't seem to replicate this error. It makes sense to have the postcss config in dev-server since we've added `importLoaders=1`, but can u show me the commands you've used? Perhaps I'm missing something.